benchmarks/ctia/entity/actor_bench.clj

```diff
@@ -13,7 +13,7 @@
 
 (defgoal create-actor "Create Actor"
   :setup (fn [] [(setup-ctia-es-store!)])
-  :cleanup (fn [_] cleanup-ctia!))
+  :cleanup (fn [{:keys [app]}] (cleanup-ctia! app)))
```

Comment:
AFAICT you have to actually execute the cleanup here https://github.com/davidsantiago/perforate/blob/61bf670d806e6696785324d89e0fcac5ffad205d/src/perforate/core.clj#L22-L23